File: app/src/main/java/in/woloo/www/woloo_host/mvp/CreateWolooHostView.java
package in.woloo.www.woloo_host.mvp;

import in.woloo.www.more.models.ViewProfileResponse;
import in.woloo.www.woloo_host.model.GeoCodeResponse;

public interface CreateWolooHostView {
    void setProfileResponse(ViewProfileResponse viewProfileResponse);
    void addWolooHostSuccess(String message);
    void geoCodeResponseSuccess(GeoCodeResponse geoCodeResponse);
}

File: app/src/main/java/in/woloo/www/more/woloo_host/mvp/CreateWolooHostView.kt
package `in`.woloo.www.more.woloo_host.mvp

import `in`.woloo.www.more.models.ViewProfileResponse
import `in`.woloo.www.more.woloo_host.model.GeoCodeResponse

interface CreateWolooHostView {
    fun setProfileResponse(viewProfileResponse: ViewProfileResponse?)
    fun addWolooHostSuccess(message: String?)
    fun geoCodeResponseSuccess(geoCodeResponse: GeoCodeResponse?)
}
